README.yaml

```diff
@@ -53,6 +53,26 @@ usage: |-
   In order to retrieve Terraform State, we configure an S3 Bucket to store plan files and a DynamoDB table to track plan metadata. Both will need to be deployed before running
   this action. For more on setting up those components, see the `gitops` component (__documentation pending__). This action will then use the [github-action-terraform-plan-storage](https://github.com/cloudposse/github-action-terraform-plan-storage) action to update these resources.
 
+  ### Config
+
+  The action expects the atmos gitops configuration file to be present in the repository in `./.github/config/atmos-gitops.yaml`.
+  The config should have the following structure:
+  
+  ```yaml
+    atmos-version: 1.45.3
+    atmos-config-path: ./rootfs/usr/local/etc/atmos/
+    terraform-state-bucket: cptest-core-ue2-auto-gitops
+    terraform-state-table: cptest-core-ue2-auto-gitops
```

Comment:
We *have to* rename these. These are not for the Terraform State Bucket. These are for the Terraform Plan File.

Reply:
@goruha https://cloudposse.atlassian.net/browse/DEV-1537